BasicGUI/Transformation.h

```diff
@@ -0,0 +1,30 @@
+
+#ifndef TRANSFORMATION_H
+#define TRANSFORMATION_H
+
+#include <QColor>
+#include <QPixmap>
+#include <QPoint>
+#include <QVector>
+#include <cmath>
+
+class Transform
+{
+public:
+    Transform(QPoint rotCenter, float deg_angle);
+    QPointF transformFrom(QPointF point);
+    QPointF operator()(QPointF point);
+
+private:
+    float static calcModule(QPointF point, float angle);
+    float static calcAngle(QPointF point);
+    float static convertAngleToRad(float deg_angle);
+    float static updateAngle(float ext_angle, float angle);
```

Comment:
Those methods are static and private. There is no benefit they are part of class.

Please move them to anonymous namespace in source file - still you will be able to use them as implementation details, and API of your class will be cleaner.

Reply:
Implemented